packages/link/src/Link.ts

```diff
@@ -99,25 +99,23 @@ function eventsListener(
             type: 'frontTransferDestinationTokens',
             payload: currentOptions.transferDestinationTokens
           },
-          iframeUrlObject?.origin || 'https://web.getfront.com'
+          iframeUrlObject?.origin || 'https://web.meshconnect.com'
         )
       }
 
       currentOptions?.onEvent?.({ type: 'pageLoaded' })
       break
     }
     default: {
-      if (isFrontEventTypeKey(event.data.type)) {
+      if (isLinkEventTypeKey(event.data.type)) {
         currentOptions?.onEvent?.(event.data)
       }
       break
     }
   }
 }
 
-export const createFrontConnection = (
-  options: FrontOptions
-): FrontConnection => {
+export const createLink = (options: LinkOptions): Link => {
   const openPopup = async (iframeUrl: string) => {
```

Comment:
Please also remove openPopup and closePopup functions from here, and from the README file as well.